crates/starknet-devnet-core/src/starknet/state_update.rs

```diff
@@ -47,13 +46,15 @@ mod tests {
         let sierra_class_hash =
             ContractClass::Cairo1(contract_class.clone()).generate_hash().unwrap();
 
-        let casm_contract_class = CasmContractClass::from_contract_class(
-            contract_class.clone(),
-            true,
-            MAX_BYTECODE_SIZE_LIMIT,
```

Comment:
Should the constant `MAX_BYTECODE_SIZE_LIMIT` be removed now that it's unused?

Reply:
[90c2edf](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/391/commits/90c2edf13ffd8d2370cc41eaf1e4a61247504828)